test/spec/adapters/criteo_spec.js

```diff
@@ -2,10 +2,54 @@
 
 import Adapter from '../../../src/adapters/criteo';
 import bidManager from '../../../src/bidmanager';
+import {ajax} from '../../../src/ajax'
 import {expect} from 'chai';
 
 var CONSTANTS = require('../../../src/constants');
 
+/* ------------ Publishertag stub begin ------------ */
+window.Criteo = {
```

Comment:
Test preconditions should be in a `before` or `beforeEach` block. See https://mochajs.org/#hooks